docs/contributing/index.md

```diff
@@ -1,69 +1,94 @@
 
-    Note that contributions that ignore guidelines are not contributions but nuisances.
+### Contributing to open source projects
 
-## Contribution workflows
+If you are new to open source projects, we recommend reading GitHub's [How to Contribute to Open Source guide](https://opensource.guide/how-to-contribute/) for an overview of why people contribute to open source projects, what it means to contribute and much more besides.
 
-There are several types of contributions and the workflows differ between the types.
-See the description for your contribution type.
+## How can I get help?
+
+Do not open issues for general support questions as we want to keep GitHub issues for confirmed bug reports. Instead, open a discussion in the Q&A category. For more details on our support mechanisms and links to troubleshooting guides, refer to our [Support guidelines](../support/index.md).
+
+!!! note
+
+    Issues created for questions or unconfirmed bugs will be moved to GitHub discussions by the maintainers.
+
+## How can I contribute?
 
 ### Discussions
 
-You can contribute to Autoware by facilitating discussions:
+You can contribute to Autoware by facilitating and participating in discussions, such as:
+
+- [Proposing a new feature to enhance Autoware](https://github.com/orgs/autowarefoundation/discussions/categories/feature-requests).
+- [Joining an existing discussion and expressing your opinion](https://github.com/orgs/autowarefoundation/discussions).
+- Organizing discussions for other contributors.
+- [Answering questions and supporting other contributors](https://github.com/autowarefoundation/autoware/discussions/categories/q-a?discussions_q=category%3AQ%26A+is%3Aunanswered).
+
+### Working groups
 
-- Find a bug and open a new issue.
-- Propose an idea of features that will enhance Autoware.
-- Join an existing discussion and express your opinion.
-- Organize discussions for other contributors.
-- Answer questions and support other contributors.
+The [various working groups](https://github.com/autowarefoundation/autoware-projects/wiki#working-group-list) within the Autoware Foundation are responsible for managing low level activities as directed by the foundation's Technical Steering Committee. These working groups are open to everyone, and joining a particular working group will allow you to gain an understanding of current projects, see how those projects are managed within each group and contribute to issues that will help progress a particular project.
 
-See our [support guidelines](../help/support-guidelines.md) for the detailed steps.
+To see the schedule for upcoming working group meetings, refer to the [Autoware Foundation events calendar](https://calendar.google.com/calendar/u/0/embed?src=autoware.org_6lol0ho5ft0217h8c60pi1fm30@group.calendar.google.com).
+
+### Reporting bugs
```

Comment:
To be consistent with other headers.
Or you can change other headers like `Joining working groups`.

```suggestion
### Bug reports
```